plugins/controlboard/src/ControlBoard.cc

```diff
@@ -254,5 +266,67 @@ GZ_REGISTER_MODEL_PLUGIN(GazeboYarpControlBoard)
             }
             return;
         }
+        #else
+        yarp::os::Bottle driver_group;
+        yarp::os::Bottle virt_group;
+
+        m_useVirtAnalogSensor = m_parameters.check("useVirtualAnalogSensor", yarp::os::Value(false)).asBool();
+        if (m_useVirtAnalogSensor)
+        {
+            virt_group = m_parameters.findGroup("VIRTUAL_ANALOG_SERVER");
+            if (virt_group.isNull())
+            {
+                yCError(GAZEBOCONTROLBOARD) <<"[VIRTUAL_ANALOG_SERVER] group not found in config file";
+                return;
+            }
+
+            yarp::os::Bottle& robotName_config = virt_group.addList();
+            robotName_config.addString("robotName");
+            robotName_config.addString(m_robotName.c_str());
+        }
```

Comment:
I guess that if `GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS` is TRUE, the `useVirtualAnalogSensor` is not relevant anymore, what do you think?

Reply:
Actually I don't know very well how it works, I don't know if anyone is interested to make it relevant. I think it is not used anymore also.

Reply:
If the `useVirtualAnalogSensor`  is on, it just creates an additional (implicit) network wrapper. Fot that reason I think we can just remove anything related to it if `GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS` is true.

Reply:
removed